tests/open-ce-tests.yaml

```diff
@@ -3,15 +3,23 @@ tests:
      command: |
          echo "setup"
          git clone --recursive -b r$(python -c "import tensorflow; print(tensorflow.__version__)") https://github.com/tensorflow/models
-         conda install pandas tensorflow-addons tensorflow-model-optimization
+         cd models
+         git apply ${FEEDSTOCK_DIR}/recipe/0304-Fixed-models-test-on-x86.patch
+         cd ..
+         conda install -y pandas tensorflow-addons tensorflow-model-optimization 
    - name: Run Recommendation model
      command: |
+         CURDIR=$(pwd)
+         export PYTHONPATH=$PYTHONPATH:$CURDIR/models
          cd models/official/recommendation
          mkdir data
          python movielens.py --data_dir=./data --dataset=ml-1m
          python ncf_keras_main.py --data_dir=./data --dataset=ml-1m
+         cd ../../..
```

Comment:
These `cd` commands at the end of the tests are unnecessary, since the next tests will be run in a separate shell.

Reply:
Done